helper/handler.php

```diff
@@ -66,7 +69,7 @@ public function resolveMoves($old, $type) {
 
         if($type != 'media' && $type != 'page') throw new Exception('Not a valid type');
 
-        $old = resolve_id($this->origNS, $old, false);
+        $old = (new PageResolver($this->origNS))->resolveId($old);
```

Comment:
`resolve_id` is not the same as `PageResolver`. I assume, but I'm not sure, that this breaks the tests as you can see from the checks. The test breakages look quite strange, as if there was a quite fundamental error somewhere. I don't know where it comes from but please run the tests and debug where the differences come from.

Reply:
Changed from PageResolver to FileResolver.